### PR TITLE
Add a CI run with JDK 12

### PR DESCRIPTION
**Pull request by gsmet**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -159,6 +159,16 @@ stages:
...
 
+      - job: Build_JDK13_Linux
+        timeoutInMinutes: 60
+        displayName: 'Linux JDK12 Build'
+        pool:
+          vmImage: 'Ubuntu 16.04'
+        steps:
+          - template: ci-templates/jvm-build-steps.yaml
+            parameters:
+              jdk: '1.12'
```
Comment (geoand) [inline azure-pipelines.yml]: I don't think this exists on Azure - or at least it didn't last time I checked

File: azure-pipelines.yml [hunk 2]
```diff
@@ -159,6 +159,16 @@ stages:
             parameters:
               jdk: '1.11'
 
+      - job: Build_JDK13_Linux
```
Comment (geoand) [inline azure-pipelines.yml]: Wrong name?

File: azure-pipelines.yml (hunk 1 above)
Comment (gsmet) [reply azure-pipelines.yml]: It is now. I got it to run and had a test failure due to a real issue with JDK 12 in one of our tests.

Comment (geoand) [reply azure-pipelines.yml]: Oh, cool!

File: azure-pipelines.yml (hunk 2 above)
Comment (gsmet) [reply azure-pipelines.yml]: Ah missed it. I targeted JDK 13 at first (just to know if it was already available on CI) and forgot to rename this one. Fixed.

**Review by geoand — APPROVED** [review]

**Comment by geoand** [comment]
@gsmet I'll let you add the milestone you believe is more applicable :)

**Comment by Sanne** [comment]
Did noone start with JDK13 yet?

**Comment by gsmet** [comment]
I plan to run it this evening once I’m done. 

It’s not available on Azure, that’s why I used 12 for now. We will upgrade to 13 ASAP.

**Pull request merged**